content/participants/ttjc@DungeonMaster976/index.md

```diff
@@ -2,26 +2,26 @@
 title: Samruddha Prabhu
 type: "participant"
 name: "Samruddha Prabhu"
-portfolio: "https://"
+portfolio: "will update"
 github-link: "https://github.com/DungeonMaster976"
 linkedin-link: "https://www.linkedin.com/in/Samruddha-prabhu-78276314a/"
 looking-for: "job / internship"
```

Comment:
Hey! @DungeonMaster976 You can either write job or internship here.

Reply:
@khusharth @DungeonMaster976 both are accepted can write job/internship.